src/webportal/src/app/home/home/virtual-cluster-statistics.jsx

```diff
@@ -97,12 +98,12 @@ const vcListColumns = [
   {
     key: 'detail',
     minWidth: 200,
-    name: 'Detail',
+    name: 'Detail (Used / Total - Bad)',
```

Comment:
>name: 'Detail (Used / Total - Bad)', [](start = 4, length = 36)

I mean:

name: 'Detail: Used / (Total - Bad)' ?

Reply:
what if there's no bad? seems there's no agreement on UI design yet

Reply:
And always use  the format:
such as 1 / (10 - 0)

if u use insufficient, it will be:
1 / (10)

Which is inconsistent with the header.

---
In reply to: [399080739](https://github.com/microsoft/pai/pull/4329#discussion_r399080739) [](ancestors = 399080739)

Reply:
no bad = bad is 0